code/game/machinery/Sleeper.dm

```diff
@@ -272,7 +272,7 @@
 				if(beaker.reagents.total_volume < beaker.reagents.maximum_volume)
 					var/pumped = 0
 					for(var/datum/reagent/x in occupant.ingested.reagent_list)
-						occupant.reagents.trans_to_obj(beaker, 3)
+						occupant.ingested.trans_to_obj(beaker, 3)
 						pumped++
 					if(ishuman(occupant))

```

Comment:
You can only put humans in sleepers, what's this doing here and why do humans get pumped twice?

Reply:
I was copying how it works a few lines above.

Reply:
The lines above take some of the patient's blood from the `vessel` reagent container, it doesn't pump their reagents again.

Reply:
resolved